x-pack/plugins/security_solution/public/common/components/event_details/osquery_tab.tsx

```diff
@@ -38,50 +63,56 @@ export const useOsqueryTab = ({
   }
 
   const { OsqueryResults } = osquery;
-  const parameters = rawEventData.fields['kibana.alert.rule.parameters'];
+  const expandedEventFieldsObject = expandDottedObject(
+    rawEventData.fields
+  ) as ExpandedEventFieldsObject;
```

Comment:
nit: can avoid the cast by making kibana an optional field in ExpandedEventFieldsObject and then annotating the type of this 
```  
const expandedEventFieldsObject: ExpandedEventFieldsObject = expandDottedObject(
    rawEventData.fields
);
```